cmd/agent/core/agent.go

```diff
@@ -187,7 +187,6 @@ func run(c *cli.Context, backends []types.Backend) error {
 		"hostname": hostname,
 		"platform": engInfo.Platform,
 		"backend":  backendEngine.Name(),
-		"repo":     "*", // allow all repos by default
```

Comment:
Might be a breaking change as agent owners were allowed to filter for repos as well. We could solve this by adding the new enforced filters with sth like `internal_repo_id` and `internal_org_id` and leave `repo` untouched